Makefile

```diff
@@ -3,4 +3,4 @@
 main:
 	@npm run build
 publish:
```

Comment:
Maybe just get rid of `publish` altogether until if/when we decide we are actually going to publish something to npm

Reply:
Ahh good point